### PR TITLE
Fet/product info logic

### PR DESCRIPTION
**Pull request by Hello07Anas**

### CODE REVIEW

File: SwiftCart/Scenes/ProductInfo/ViewModel/ProductInfoVM.swift [hunk 1]
```diff
@@ -10,14 +10,27 @@ import Foundation
 class ProductInfoVM {
     
     typealias Product = Any
```
Comment (CSAhmedOsman) [inline SwiftCart/Scenes/ProductInfo/ViewModel/ProductInfoVM.swift]: you need to delete this line

Comment (Hello07Anas) [reply SwiftCart/Scenes/ProductInfo/ViewModel/ProductInfoVM.swift]: consider it done ^_*

**Comment by Hello07Anas** [comment]
Done @CSAhmedOsman

**Review by CSAhmedOsman — APPROVED** [review]
That looks good, nice work!

**Pull request merged**